layouts/partials/article.html

```diff
@@ -27,6 +27,17 @@
     {{ $content = len .Content }}
 {{ end }}
 
+<script>
+    function copyPermalink(link) {
+        // Copy the article link to the clipboard
+        let sanitizedLink = link;
+        while (sanitizedLink[0] === '/') {
+            sanitizedLink = sanitizedLink.substring(1);
+        }
+        navigator.clipboard.writeText(sanitizedLink);
+    }
+</script>
```

Comment:
This whole script should be moved to the `baseof` layout, there's no sense in repeating it for each article

Reply:
Great point. Thanks. I moved it.